lightning/src/onion_message/blinded_route.rs

```diff
@@ -13,9 +13,11 @@ use bitcoin::secp256k1::{self, PublicKey, Secp256k1, SecretKey};
 
 use chain::keysinterface::KeysInterface;
 use super::utils;
+use ::get_control_tlv_length;
```

Comment:
This seems bad. Will check why `#[macro_export]` places the `get_control_tlv_length` in the root of the lightning crate for `onion_message::utils`, but not for in `ln::functional_test_utils` and similar.